apps/mull-ui/src/app/pages/direct-message/direct-message.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../components';
+import ContactRow from '../../components/contact-row/contact-row';
+import './direct-message.scss';
+
+/* eslint-disable-next-line */
+export interface DirectMessageProps {}
+
+// TODO: Query that returns an array of user profiles related to user
+const userData = [
+  {
+    id: 1,
+    name: 'Bob Marley',
+    description: 'Singer',
```

Comment:
It's not the description of the user that's shown on this page, it's the last message sent by either party.